src/humancellatlas/data/metadata/api.py

```diff
@@ -204,7 +204,7 @@ def __init__(self, json: JSON):
         super().__init__(json)
         content = json.get('content', json)
         self.genus_species = {gs['text'] for gs in content['genus_species']}
-        self.disease = {d['text'] for d in content.get('disease', []) if d}
+        self.diseases = {d['text'] for d in lookup(content, 'disease', 'diseases', default=set()) if d}
```

Comment:
The new name should always be tried first.

```suggestion
        self.diseases = {d['text'] for d in lookup(content, 'diseases', 'disease', default=set()) if d}
```

Reply:
The default should be `[]`. In your previous version it would have had to be set() but here you are not providing a default value for `self.diseases` but a default value for what you extract from the JSON. JSON cannot contain sets.